ngSe/exceptions.py

```diff
@@ -47,6 +47,7 @@ class FrontEndError(Exception):
     selenium_exceptions.InvalidElementStateException,
     selenium_exceptions.NoSuchElementException,
     selenium_exceptions.ElementNotVisibleException,
+    selenium_exceptions.StaleElementReferenceException,
```

Comment:
I remember leaving this out because it made steps that were *supposed* to fail (because something was broken) take a long time to fail, because most of the time this exception meant something was being done wrong. But this isn't really an issue I guess, we should vet this against some existing code just to be safe though